### PR TITLE
Add base source tables for Fantom & Goerli

### PR DESCRIPTION
**Pull request by hildobby**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/base_sources/goerli_base_sources.yml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+          - name: hash
+            description: "Primary key of the transaction"
+            tests:
+              - not_null
+          - name: type
+            description: "Transaction type"
+          - name: access_list
+            description: "Specifies a list of addresses and storage keys"
+            description: "Specifies a list of addresses and storage keys"
+          - name: l1_block_number
```
Comment (soispoke) [inline models/base_sources/goerli_base_sources.yml:65]: @hildobby I think there are a lot of columns that were copied from the Optimism transactions table but that don't exist on Fantom here

**Review by soispoke — COMMENTED** [review]
great initiative, this will be very helpful 👍

File: models/base_sources/goerli_base_sources.yml (hunk 1 above)
Comment (hildobby) [reply models/base_sources/goerli_base_sources.yml:65]: 🫠 alright, let me fix this real quick

**Comment by soispoke** [comment]
thanks! seems like a test is failing on dupes in the fantom.transactions table, the team is working on it to resolve it but it shouldn't prevent us from merging imo (tagging @dsalv and @dot2dotseurat for visibility)

**Review by soispoke — APPROVED** [review]
lgtm

**Comment by soispoke** [comment]
I'll merge and still monitor duplicates on the `fantom.transactions` table, by checking the uniqueness test in place

**Pull request merged**